tests/ballerina-spec-conformance-tests/src/test/java/org/ballerinalang/test/TestRunnerUtils.java

```diff
@@ -196,6 +199,36 @@ private static String validateKindOfTest(String kind) {
         }
     }
 
+    public static void validateLabels(String labels, Set<String> selectedLabels) {
```

Comment:
Any reason to use the term `selectedLabels` instead of `predefinedLabels`(as per the passed argument name) here?